### PR TITLE
Lmsa 8968

### PR DESCRIPTION
**Pull request by mrw-iu**

### CODE REVIEW

File: src/main/js/components/SuccessAlert.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+ */
+import React from 'react'
+
+class SuccessAlert extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    componentDidUpdate() {
```
Comment (mrw-iu) [inline src/main/js/components/SuccessAlert.js:41]: double check eventlisteners

File: src/main/js/App.js [hunk 2]
```diff
@@ -112,14 +102,10 @@ class App extends React.Component {
     }
     return (
         <div>
-            <div className="rvt-container" id="main-container" role="main">
+            <div className="rvt-container-xl" id="main-container" role="main">
```
Comment (mrw-iu) [inline src/main/js/App.js:105]: check role=main

File: src/main/js/components/ConfirmationModal.js [hunk 3]
```diff
@@ -31,66 +31,121 @@
...
+              <h1 className="rvt-dialog__header rvt-dialog__title" id={`${this.props.dialogId}-title`} tabIndex="-1">{this.props.title}</h1>
+
+              <div className="rvt-dialog__body">
+                <div id={`${this.props.dialogId}-description`}>{this.props.children}</div>
+              </div>
+              <div className="rvt-dialog__controls">
+                <button id={`${this.props.dialogId}-yes`} key="yes" type="button" className="rvt-button loading-btn" onClick={(event) => this.confirmationActions(event, this.props.handleConfirm, this.props.showLoading, this.props.loadingText, this.props.dialogId)}>
+                  <span className="rvt-button__content">{this.props.yesLabel}</span>
+                  <span className="rvt-loader rvt-loader--xs rvt-display-none"></span>
+                  <p aria-live="polite"><span class="rvt-sr-only spinner-sr-text rvt-display-none">Loading</span></p>
```
Comment (maurercw) [inline src/main/js/components/ConfirmationModal.js:137]: className instead of class?

File: src/main/js/components/ConfirmationModal.js [hunk 4]
```diff
@@ -31,66 +31,121 @@
...
+                </button>
+                <button type="button" className="rvt-button rvt-button--secondary loading-btn" data-rvt-dialog-close={`${this.props.dialogId}-dialog`} onClick={this.props.onDismiss} >
+                  <span>{this.props.noLabel}</span>
+                </button>
+              </div>
+              <button className="rvt-button rvt-button--plain rvt-dialog__close loading-btn" data-rvt-dialog-close={`${this.props.dialogId}-dialog`} role="button"
+                onClick={this.props.onDismiss}>
```
Comment (mrw-iu) [inline src/main/js/components/ConfirmationModal.js:144]: onclick and eventlistener

File: src/main/js/components/coursetemplating/ApplyTemplate.js [hunk 5]
```diff
@@ -88,30 +87,49 @@ class ApplyTemplate extends React.Component {
...
-        this.setState({applyModalOpen: true, modalData: {templateId: templateId, templateName: templateName, nodeName: nodeName}})
+        // move focus to the dialog heading
+        var dialogHeading = $("#apply-template").find("h1.rvt-dialog__title").first();
+        dialogHeading.focus();
+
+        // When the rivet dialog closes, it always is the last event.  So when we try to manually move
+        // the focus on cancel, it gets overridden when the modal closes (the first instance of data-rvt-dialog-trigger always gets focus).
+        // To combat this, we remove the trigger from every button except the one that was just clicked. We trick rivet so it
+        // returns focus to the correct button
+
```
Comment (mrw-iu) [inline src/main/js/components/coursetemplating/ApplyTemplate.js:100]: alternative trigger setting

**Pull request merged**